packages/slice-machine/src/modules/availableCustomTypes/index.ts

```diff
@@ -105,6 +106,8 @@ export const availableCustomTypesReducer: Reducer<
       };
     }
     case getType(createCustomTypeCreator.success): {
+      action.payload.newCustomType.__status = CustomTypeStatus.New;
+
       const normalizedNewCustomType = normalizeFrontendCustomType(
         action.payload.newCustomType
       );
```

Comment:
need to update the status for the `renameCustomTypeCreator.success` redux action

Reply:
also need to update the status when `saveCustomTypeCreator.success` and `pushCustomTypeCreator.success` are called.

Right now if you push a custom type you update its status on the selectedCustomType store but not on the availableCustomTypes, so I think if you push and navigate back to the main page the status will not have updated